scripts/aggregate_tracks

```diff
@@ -29,43 +42,44 @@ TABLE_DESC=(
   "$@"
 )
 TABLE_DESC=$( IFS=$'\n'; echo "${TABLE_DESC[*]}" )
-
-
 echo "Publishing tracks to ${TRACKS_TABLE}..."
 echo "${TABLE_DESC}"
-
-echo "  Create table"
-
+SCHEMA=${ASSETS}/bigquery/tracks.schema.json
 bq mk --force \
   --description "${TABLE_DESC}" \
   --schema ${SCHEMA} \
   ${TRACKS_TABLE}
-
 if [ "$?" -ne 0 ]; then
   echo "  Unable to create table ${TRACKS_TABLE}"
   exit 1
 fi
 
-echo "  Deleting existing records for table ${TRACKS_TABLE}"
-
+################################################################################
+# Clean up existing records
+################################################################################
+echo "Deleting existing records for table ${TRACKS_TABLE}"
+DELETE_SQL=${ASSETS}/bigquery/delete.sql.j2
 jinja2 ${DELETE_SQL} -D table=${TRACKS_TABLE//:/.} \
      | bq query --max_rows=0
-
 if [ "$?" -ne 0 ]; then
   echo "  Unable to delete records for table ${TRACKS_TABLE}"
   exit 1
 fi
 
+################################################################################
+# Inserting new records
+################################################################################
 echo "  Inserting new records for table ${TRACKS_TABLE}"
-
+INSERT_SQL=${ASSETS}/bigquery/aggregate-tracks.sql.j2
 jinja2 ${INSERT_SQL} \
```

Comment:
Just a thought... I usually do this in a two step process:
```
SQL=jinja2 ${INSERT_SQL}
echo $SQL
echo $SQL | bq query
```
That way I have the query in the log for debugging purposes.